test/e2e/opm_test.go

```diff
@@ -425,6 +431,91 @@ var _ = Describe("opm", func() {
 			err = buildIndexWith(containerTool, indexImage, nextIndex, bundles[4:].images(), registry.ReplacesMode, true) // 1.0.1-overwrite
 			Expect(err).NotTo(HaveOccurred())
 		})
+
+		It("is consistent with existing update graphs", func() {
```

Comment:
This looks good. I'm wondering if this can take the place of #463 a PR I had open for a while that attempts to do something similar but didn't really go anywhere

Reply:
oh cool I didn't realize you had started that! I'll take a look to see what I can steal from there :) thanks!

Reply:
That's a good idea -- I think this PR goes one step further by actually running integration tests with the index, my original PR just checks to see that `opm registry serve <index>` works. I think closing my original PR when this one merges makes sense.